osu.Game.Rulesets.Catch/Skinning/Legacy/CatchLegacySkinTransformer.cs

```diff
@@ -66,7 +66,7 @@ public override Drawable GetDrawableComponent(ISkinComponent component)
                         return null;
 
                     case CatchSkinComponents.Catcher:
-                        var version = Source.GetConfig<LegacySkinConfiguration.LegacySetting, decimal>(LegacySkinConfiguration.LegacySetting.Version)?.Value ?? 1;
+                        var version = GetConfig<LegacySkinConfiguration.LegacySetting, decimal>(LegacySkinConfiguration.LegacySetting.Version)?.Value ?? 1;
```

Comment:
Seems a bit weird that this call specifically is not calling on `Skin` but `this`. Might want to add a note here if that's done for a specific reason.

Reply:
More like best practice in transformers, if you're calling a method that's not this method, then call it with `this`, otherwise call `Skin`.

A bit of confusing one, but that's that. And not sure if a note is worth really.

Reply:
I've applied the changes proposed here (https://github.com/ppy/osu/pull/13423/commits/fbb856d84bfdfb2e5c4c5ee3fbfde90302864000). Interestingly though, previously some transformers were not calling `base.GetSample` and instead calling `Skin.GetSample` directly, this makes them skip certain logic in the base sample lookup method regarding overriding layered hit samples with virtual samples if they're turned off in skin settings...

I guess it's more correct now? might have to check stable for this one.